src/web/src/components/Login/LoggedOut.tsx

```diff
@@ -1,6 +1,6 @@
 import ExitToAppIcon from '@material-ui/icons/ExitToApp';
 import config from '../../config';
-import HeaderButton from '../Header/HeaderButton';
+import HeaderButton from '../NavBar/NavBarButton';
```

Comment:
Since we are using NavBar, let's use NavBarButton and NavBarIcon as well.
Sorry for only cath this now.